src/Libraries/RevitNodes/Application/Document.cs

```diff
@@ -62,6 +63,33 @@ public static Document Current
             get { return new Document(DocumentManager.Instance.CurrentDBDocument); }
         }
 
+        /// <summary>
+        /// Gets the worksharing path of the current document
+        /// </summary>
+        public string GetWorksharingPath
```

Comment:
If this is a query node and we're just returning a property, we could just call the node `WorksharingPath` ?